content/docs/command-reference/commit.md

```diff
@@ -279,5 +279,6 @@ $ dvc status
 Data and pipelines are up to date.
 ```
 
-Nothing special is required, we simply `commit` to both the SCM and DVC. Since
-this pipeline is up to date, `dvc repro` will not do anything.
+Instead of reproducing the pipeline for changes that do not produce a different
+output of pipeline stages, use `commit` to both git and DVC. Since this pipeline
+is up to date, `dvc repro` will not do anything.
```

Comment:
Let's just simplify/generalize terminology, so complex context that aren't closely related to this cmd ref or example are avoided:
```suggestion
Instead of reproducing the pipeline for changes that do not produce a different
results, just use `commit` on both Git and DVC.
```
That last sentence was still redundant vs the previous paragraph before the code block...

Reply:
But this is kind of still missing the main point I mentioned in https://github.com/iterative/dvc.org/pull/1327#discussion_r429603929

Reply:
> It's something along the lines of being able to force-register the changes to a stage/pipeline without reproducing it.

Reply:
p.s. the keyword being "force"

Reply:
I thought we would use the force option when committing to both DVC and Git if force pushing the changes was a requirement but it was not used in the example given. I am sorry, I think I got confused.

Reply:
Hmmm I forgot there's a `--force` option. I guess using the term "force" here could be confusing after all. It's not really related to force-pushing to Git though.

But my point is the main idea for this last paragraph should be or should include that we are "forcing" DVC to commit the stage file changes even when the stage wasn't reproduced after it's dependencies change (because we know there's no need). Did you actually run the example commands to fully understand the example? In fact it may even need the `--force` option, I'm not sure. Please double check 🙂

Reply:
Yes I ran the commands. By using the -f option the prompt doesn't ask the user a yes/no question regarding the decision of committing. That is the only difference. Personally, I don't think it is necessary to put the force option but if that provides more clarity then we can do it.

Reply:
I think it looks good already, the motivation for using commit in this example is provided a bit above - "we don't want to run repro because changes are not substantial ...".

(also, improving example itself probably out of scope for the ticket)